src/components/layout/TabScrollScreen.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
```

Comment:
The only difference between this component and `ScrollScreen` is that it scrolls to the top when blurred? 
If yes, should we maybe keep only 1 configurable `ScrollScreen`?

Reply:
Mhh, we could just use the code from `TabScrollScreen` (renaming the component `ScrollScreen`), and simply check the condition (`scrollToTopOnBlur`) in the useEffect? 

One other issue I have with this `TabScrollScreen` is : "how is its implementation related to Tabs"? :)